scripts/settings/save-from-env.js

```diff
@@ -116,7 +117,13 @@ const updateTokens = async ({
     await NetworkTokensModel.findOrCreate({
       where: {
         tokenId: token.id,
-        networkId: beproNetwork.id
+        networkId: beproNetwork.id,
+      },
+      defaults: {
+        tokenId: token.id,
+        networkId: beproNetwork.id,
+        isTransactional: true,
+        isReward: false
```

Comment:
`updateTokens` has a `isTransactional` prop, we can use it to define when it is reward or not, and when the relation already exists we need to update `isTransactional` and `isReward` according to this prop